src/main/java/Main.java

```diff
@@ -52,16 +58,14 @@ public static void main(String[] args) {
         System.exit(0);
     }
 
-
-
     private void getStartedDemo() throws Exception {
         System.out.println("Using Azure Cosmos DB endpoint: " + AccountSettings.HOST);
 
         client = new AsyncDocumentClient.Builder()
```

Comment:
We should not be using any implementation classes for this sample. (All the classes from the package `implementation.*`)

There are public API classes now such as CosmosClientBuilder which builds CosmosClient. You should be able to perform a queryContainers then do a foreach loop to exhaust all the results. There is no need to manage the continuation token.

https://azuresdkdocs.blob.core.windows.net/$web/java/azure-cosmos/latest/com/azure/cosmos/CosmosDatabase.html#queryContainers-java.lang.String-